### PR TITLE
Handle routing information for each bulk item in the elastic connector

### PR DESCRIPTION
**Pull request by mfelsche**
# Pull request

## Description

Previously we didnt handle routing metadata for single bulk items in a batched event, now we do.

## Checklist

* [x] The RFC, if required, has been submitted and approved
* [x] Any user-facing impact of the changes is reflected in docs.tremor.rs
* [x] The code is tested
* [x] Use of unsafe code is reasoned about in a comment
* [x] Update CHANGELOG.md appropriately, recording any changes, bug fixes, or other observable changes in behavior
* [x] The performance impact of the change is measured (see below)

### CODE REVIEW

**Review by Licenser — DISMISSED** [review]
🚀

File: src/connectors/impls/elastic.rs [hunk 1]
```diff
@@ -703,101 +704,156 @@ impl<'a, 'value> ESMeta<'a, 'value> {
...
+        Ok(())
+    }
+
+    fn insert_update_op(&self, data: &Value, ops: &mut BulkOperations) -> Result<()> {
+        let mut op = self
+            .get_id()
+            .map(|id| {
+                // we are sure that this BulkUpdateOperation lives only as long as the event
+                // so it is fine to only do a `.clone()` here instead of a `.clone_static()`
+                let src = literal!({ "doc": data.clone() });
```
Comment (mfelsche) [inline src/connectors/impls/elastic.rs:819]: this one is indeed serialized as soon as it is pushed below, so this is *totally* safe :P

**Review by Licenser — APPROVED** [review]

**Review by Licenser — APPROVED** [review]
I think we also need a docs update to indicate that we now have `routing` as an option

**Comment by mfelsche** [comment]
It already says it is there in the docs, thats why this is a bug.

**Comment by Licenser** [comment]
Oh sneaky so the docs PR was done long before the code PR :joy: more of a very long lag between code and docs then :P

**Pull request merged**